backend/testClient.spec.js

```diff
@@ -0,0 +1,216 @@
+const { createTestClient } = require('apollo-server-testing');
+
+const { typeDefs } = require("./typeDefs");
+const { resolvers } = require("./resolvers");
+
+const { ApolloServer, gql } = require('apollo-server');
+
+const { jwt } = require('jsonwebtoken');
+
+const { v1 } = require('neo4j-driver');
+
+const { makeAugmentedSchema } = require('neo4j-graphql-js');
+const schema = makeAugmentedSchema ({ typeDefs , resolvers });
+
+let token;
+let query;
+let mutate;
+
+describe("ServerTest", () => {
+
+  beforeAll(async () => {
+    const neoDriver = v1.driver(
+      'bolt://localhost:7687',
+      v1.auth.basic('neo4j', 'password')
+    );
+
+    const server = new ApolloServer({
+      schema
+    });
+
+    server.requestOptions = {
+      context() {
+        return {
+          token: token,
+          neoDriver
+        }
+      }
+    };
+
+    query = createTestClient(server).query;
+    mutate = createTestClient(server).mutate;
+  });
+
+  it('test Login', async () => {
```

Comment:
Write an English sentence in simple present:
```suggestion
  describe('login', () => {
    it('returns a valid json web token', async () => {
  })
```
> it returns a valid json web token

sounds so much better than

> it test Login

doesn't it?